Runtime/Core/CinemachineCore.cs

```diff
@@ -72,7 +72,11 @@ public static CinemachineCore Instance
         /// <summary>Delegate for overriding Unity's default input system.
         /// If you set this, then your delegate will be called instead of
         /// System.Input.GetAxis(axisName) whenever in-game user input is needed.</summary>
+#if ENABLE_LEGACY_INPUT_MANAGER
         public static AxisInputDelegate GetInputAxis = UnityEngine.Input.GetAxis;
+#else
+        public static AxisInputDelegate GetInputAxis = delegate { return 0; };
```

Comment:
Did you test this?  I am unfamiliar with this syntax.

Reply:
Yes, I tested it.
What did you have in mind instead?

Reply:
I could also do this:
`public static AxisInputDelegate GetInputAxis = axisName => 0;`

But to me the delegate version was clearer.

Reply:
The OrbTransposer in CameraPositionTest tests this indirectly, when the legacy input system is disabled.

Reply:
No worries.  Like I said, I was unfamiliar with the syntax so wanted to make sure it was tested.